Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM node:6-onbuild
+EXPOSE 3000
```

Comment:
@int128  
Can you check if the comments from @zephyrweb can be incorporated so that I can merge it?